content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -607,7 +605,7 @@ So a webhook response to add that label would be:
     "uid": "<value from request.uid>",
     "allowed": true,
     "patchType": "JSONPatch",
-    "patch": "W3sib3AiOiAiYWRkIiwgInBhdGgiOiAiL3NwZWMvcmVwbGljYXMiLCAidmFsdWUiOiAzfV0="
+    "patch": "[{\"op\": \"add\", \"path\": \"/spec/replicas\", \"value\": 3}]"
```

Comment:
This change is incorrect... returning this JSON from a webhook would fail to unmarshal. The API does require base64-encoded content. When you are using go types, you set a `[]byte` directly and the base-64 encoding is done for you automatically, but we need to document the base64 requirement as seen on the wire.

If you want to add a note that when using the go types, the base-64 encoding is done for you automatically, that could be helpful.